src/Microsoft.ServiceFabric.Services.Remoting/Builder/ProxyBase.cs

```diff
@@ -39,16 +40,20 @@ internal IServiceRemotingMessageBodyFactory ServiceRemotingMessageBodyFactory
         }
 
 #if !DotNetCoreClr
+        [Obsolete("This method is part of the deprecated V1 service remoting stack. To switch to V2 remoting stack, refer to:")]
         internal abstract DataContractSerializer GetRequestMessageBodySerializer(int interfaceId);
 
+        [Obsolete("This method is part of the deprecated V1 service remoting stack. To switch to V2 remoting stack, refer to:")]
         internal abstract DataContractSerializer GetResponseMessageBodySerializer(int interfaceId);
 
         internal abstract object GetResponseMessageBodyValue(object responseMessageBody);

```

Comment:
It looks like the `GetResponseMessageBodyValue` and `CreateRequestMessageBody` methods are part of the V1 stack as well. Should we make them Obsolete as well?